dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/internal/localization/i18n/ExceptionLocalizationResource.nlsprops

```diff
@@ -433,6 +433,8 @@ unknown_bean_class=Unknown Entity bean class: {0}, please verify that this class
 
 unknown_entitybean_name=Unknown Entity Bean name: {0}
 
+unsupported_classfile_version=Class {0} was compiled with unsupported JDK. Report this error to the EclipseLink team, please.
```

Comment:
Change "Class {0} was compiled with unsupported JDK." to "The {0} class was compiled with an unsupported JDK.".
Change "Report this error to the EclipseLink team, please." to "Report this error to the EclipseLink open source project.".